### ISSUE TITLE
Watch app in development

### ISSUE THREAD

**Issue by AndreCapelache** [trailsjs/trails#105]
Any way to reload (models, controllers, services, polices, trailpacks, ...)  without restart the server in development

**Comment by tjwebb** [comment]
Not yet, but we've started working on this and definitely want to make it happen. There are two sides to this: 

1. The ability to reload/hotswap trailpacks is started, and initial implementations are here: https://github.com/trailsjs/trailpack/blob/master/index.js#L81-L111

2. We want to build a "trailpack-hotswap" that will watch directories and re-load assets that change. Trails' event-based loading system should make this a natural fit. 

So, in short, this is something we're looking at, but work by the core team likely won't begin until after the 1.0 release next month.

**Comment by tjwebb** [comment]
@AndreCapelache check out the [**Autoreload**](https://github.com/trailsjs/trailpack-autoreload) Trailpack.

**Comment by AndreCapelache** [comment]
Thanks
